src/pages/index.js

```diff
@@ -77,7 +76,7 @@ const IndexPage = ({data}) => {
         <p className="h4 mt-4">
           Learn how to <strong>electrify your own machines</strong> and <strong>pass local policy</strong> to electrify the rest
         </p>
-        <Link className='btn btn-lg btn-success mt-4' to='/take-action'>Take Action</Link>
```

Comment:
Also cleaned up using single quotes for attributes. I'm surprised the linting doesn't weight in here 🤷🏻‍♂️